config/webpack/webpack.config.js

```diff
@@ -210,8 +214,9 @@ const defaultWebpackConfig = (env = {}, argv = {}) => {
 
   const resolveModules = ['node_modules'];
 
+  let supportedLocales;
```

Comment:
Should this default be []?

Reply:
It could be, but I don't think it would provide any additional value as opposed to undefined unless we can guarantee each consuming team is using >=  the version these changes will release on.